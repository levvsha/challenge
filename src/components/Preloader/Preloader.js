/* Preloader from http://tobiasahlin.com/spinkit/ by @tobiasahlin */

import './Preloader.styl';

import React from 'react';

const Preloader = () => (
  <div className="c-preloader">
    <div className="spinner">
      <div className="bounce1" />
      <div className="bounce2" />
      <div className="bounce3" />
    </div>
  </div>
);

// Preloader.propTypes = {};

export default Preloader;
