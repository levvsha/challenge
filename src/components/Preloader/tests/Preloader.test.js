import React from 'react';
import renderer from 'react-test-renderer';

import Preloader from '../';

describe('<Preloader />', () => {
  it('Preloader renders correctly', () => {
    const tree = renderer.create(<Preloader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
