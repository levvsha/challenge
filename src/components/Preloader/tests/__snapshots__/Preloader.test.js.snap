// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preloader /> Preloader renders correctly 1`] = `
<div
  className="c-preloader"
>
  <div
    className="spinner"
  >
    <div
      className="bounce1"
    />
    <div
      className="bounce2"
    />
    <div
      className="bounce3"
    />
  </div>
</div>
`;
